### PR TITLE
Add more information for securing API Discovery endpoints

### LINKED ISSUE #198: Describe that API Discovery Endpoint don't have to be secured
Currently we are not precise how the API Discovery endpoint have or don't have to be secured.

Add following points:
- the can be both unsecured if this is suitable for given domain and product (has to be aligned with product)
- if you secure them, only use uid scope

### PR DESCRIPTION
**Pull request by fmueller**
Fixes #198

### CODE REVIEW

File: api-operation/ApiOperation.md [hunk 1]
```diff
@@ -2,41 +2,41 @@
...
-All service applications must support access to the OpenAPI Reference Definitions of their external APIs — it 
+All service applications must support access to the OpenAPI Reference Definitions of their external APIs — it
 is optional for internal APIs — via the following two API endpoints:  
 
 * endpoint(s) for GET access on its OpenAPI definition(s), for instance
   `https://example.com/swagger.json` or `https://example.com/swagger.yaml`.
-* “Twintip” discovery endpoint `https://example.com/.well-known/schema-discovery` that delivers
+* API Discovery endpoint `https://example.com/.well-known/schema-discovery` that delivers
   the OpenAPI definition endpoint(s) above (see the link below for a description of its format).
+* if these endpoints have to be secured by OAuth depends on context of your API and product management
```
Comment (ePaul) [inline api-operation/ApiOperation.md]: `if` → `whether` seems clearer here.

**Comment by ePaul** [comment]
Other than this minor issue it looks fine.
Do we need a changelog entry?

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by fmueller** [comment]
@ePaul I don't think that we need a changelog entry for such a tiny change.

@ePaul @maxim-tschumak Can both of you please re-thumb? :)

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**